### PR TITLE
Option to save two shards for low mastery champions

### PR DESCRIPTION
**Pull request by afobarlo**
"Should we keep two shards when you're low mastery on the champion?"

Keeps two shards for champion masteries lower than 5, check it out.

### CODE REVIEW

**Comment by Querijn** [comment]
Maybe it would be better as a followup question to the other "keep two shards" question?

I would also like it to be specific about what low mastery is, so maybe rephrase the question to "Should we keep two shards when you're mastery level 4 or below on the champion?"

**Comment by Querijn** [comment]
Maybe it would be better as a followup question to the other "keep two shards" question?

**Comment by afobarlo** [comment]
Yeah thats better
